koku/masu/database/trino_sql/reporting_azurecostentrylineitem_daily_summary.sql

```diff
@@ -25,8 +25,8 @@ WITH cte_line_items AS (
         resourcelocation as resource_location,
         coalesce(nullif(servicename, ''), metercategory) as service_name,
         json_extract_scalar(json_parse(additionalinfo), '$.ServiceType') as instance_type,
-        cast(coalesce(nullif(quantity, 0), usagequantity) as DECIMAL(24,9)) as usage_quantity,
-        cast(coalesce(nullif(costinbillingcurrency, 0), pretaxcost) as DECIMAL(24,9)) as pretax_cost,
+        cast(coalesce(nullif(quantity, 0), 0) as DECIMAL(24,9)) as quantity,
+        cast(coalesce(nullif(costinbillingcurrency, 0), 0) as DECIMAL(24,9)) as costinbillingcurrency,
         coalesce(nullif(billingcurrencycode, ''), nullif(currency, ''), billingcurrency) as currency,
```

Comment:
```suggestion
        coalesce(nullif(billingcurrencycode, ''), billingcurrency) as currency,
```
We should also be able to eliminate the `currency` from this select.